typha/fv-tests/state_recorder.go

```diff
@@ -25,8 +28,13 @@ import (
 )
 
 func NewRecorder() *StateRecorder {
+	return NewRecorderChanSize(1000)
```

Comment:
Decoupled the recorder onto its own thread so that I could more easily measure performance.